content/blog/news/morganepostlinkedin.fr.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Forum DGEX Solutions"
+linkTitle: "Forum DGEX Solutions"
+date: 2023-04-12
+---
+## Forum DGEX Solutions 
+
+Voici un extrait de la conférence de Loïc Hamelin au forum DGEX Solutions , au sujet du calcul automatique de sillon de dernière minute.
+
+### Mais tout d’abord qu’est ce qu’un sillon ?
+Il s’agit de la réservation d’espace/temps sur le réseau, par les entreprises ferroviaires (EF), comme SNCF Voyageurs. Ces réservations se font des années en amont (A-3). Les EF peuvent également faire des demandes de sillons en urgence (appelés sillons de dernière minute), de A-1 jusqu’au jour J.
+Ces demandes sont actuellement gérées par des guichets, et les sillons sont tracés à la main. Il y a donc un risque d’erreur humaine, et les demandes sont parfois traitées en plusieurs jours.
+
+### Qu’est ce que nous proposons ?
+Le logiciel OSRD, en plus de disposer d’un simulateur ferroviaire pour la planification des horaires, va également permettre de traiter les demandes de sillon de dernière minute de façon automatisée ! Il sera possible en seulement quelques clics d’exprimer une demande de sillon de dernière minute et obtenir une réponse en quelques secondes (tout au plus 3min) !
```

Comment:
`en quelques secondes (tout au plus 3min)` is quite weird, we are promising a computing time that varies from 2 orders of magnitude.